ramls/patron-notice-policies.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of patron notice policies",
+  "type": "object",
+  "properties": {
+    "patronNoticePolicies": {
+      "description": "List of items",
```

Comment:
List of patron notice policies

Reply:
changed to "List of patron notice policies"